modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/jaxrs/exception/DuplicateSXPBlueprintExternalReferenceCodeExceptionMapper.java

```diff
@@ -47,7 +47,10 @@ protected Problem getProblem(
 			duplicateSXPBlueprintExternalReferenceCodeException) {
 
 		return new Problem(
-			Response.Status.BAD_REQUEST, "Duplicate external reference code");
+			null, Response.Status.BAD_REQUEST,
```

Comment:
I'm assuming there's nothing useful in duplicateSXPBlueprintExternalReferenceCodeException if fill in for the detail parameter?

Reply:
Yep, I was thinking that maybe could be a good idea to use it in the FE instead of the message, it could be clean. something like:

`responseContent.type.includes("duplicateSXPBlueprintExternalReferenceCodeException")`

Or I could simply remove the parameter and keep it simple.

Reply:
After you answer that I will push my changes